app/dashboard/src/hooks/eventCallbackHooks.ts

```diff
@@ -1,23 +1,19 @@
-/**
- * @file useEventCallback shim
- */
-
-import * as React from 'react'
+/** @file `useEventCallback` shim */
+import { useCallback } from 'react'
 
-import * as syncRef from '#/hooks/syncRefHooks'
+import { useSyncRef } from '#/hooks/syncRefHooks'
 
 /**
- * useEvent shim.
+ * `useEventCallback` shim.
```

Comment:
it's a `useEvent` shim, and _should_ be called `useEvent`, but `useEvent` already exist in the project 😊